### DISCUSSION TITLE
Incorrect "Used by"

### DISCUSSION THREAD

**Discussion by bczhc** [community/community#54414] [Repositories]
### Select Topic Area

Bug

### Body

In this repository https://github.com/bczhc/librime-rs, in the "Used by" panel it shows:
![image](https://user-images.githubusercontent.com/49330580/236116011-7c22bddf-489f-4964-96f3-660f6127db65.png)

Then I took a look at repository [wlh320/rime-ls](https://github.com/wlh320/rime-ls), finding it's using another crate from repo lotem/librime-sys, with the same Rust crate name with mine.
![image](https://user-images.githubusercontent.com/49330580/236116140-13a08894-35ac-468c-95f3-01e713ed7f06.png)

BTW the crate name [`librime-sys`](https://crates.io/crates/librime-sys) on crates.io has been taken by bczhc/librime-rs. Anyway because wlh320/rime-ls is not actually using that crate from crates.io, it's unexpected to show these two repositories in bczhc/librime-rs "Used by", but should in lotom/librime-sys.

**Comment by jtlapp** [comment]
I'm having a similar problem and GitHub support informed me that if a repo ever used another repo, then that other repo will forever list the first repo as using it.

**Comment by gildas-lormeau** [comment]
This bug also appeared today on https://github.com/gildas-lormeau/zip.js. The dependencies of "@trigrou/zip-js" are displayed by default instead of "@zip.js/zip.js".

**Comment by Jenesius** [comment]
Hi all. I noticed that **Used By** was no longer displayed on my repository page. After comparing with other repositories, I noticed that when I go to the **/network/dependents** page, I get statistics for another package:
![image](https://github.com/community/community/assets/21982550/9ff76f65-ce7b-455d-9728-bf017e372900)
I think this is the main problem. How can I fix this?
Мой репозиторий: [https://github.com/Jenesius/vue-modal](https://github.com/Jenesius/vue-modal)

[The same problem here.](https://github.com/orgs/community/discussions/63362)

**Comment by Jenesius** [comment]
Solution [here](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package).

**Reply by jtlapp** [reply]
This requires asking the repo owner to make the correction. My guess is that it is not generally in their interest to reduce the number of "used by" repos appearing in this list.

I'm thinking that if GitHub cannot show an accurate list of dependent repos, then it should not show that list.

**Comment by ameligrana** [comment]
I noticed that many programming language repositories have the same issue, look at

https://github.com/JuliaLang/julia
https://github.com/rust-lang/rust
https://github.com/JetBrains/kotlin
https://github.com/golang/go
https://github.com/vlang/v
https://github.com/ruby/ruby

(I even see that some of the accounts on these repositories are the same)

Can this section be disabled so that to prevent these issues?

**Comment by ameligrana** [comment]
All of them display non-sense values as 

![immagine](https://github.com/community/community/assets/68152031/a40cf56a-eeb8-425d-a4cc-41888f995346)

**Comment by ashvardanian** [comment]
I've also got a little feedback about the "Used By" counter on repos. I've got this package, USearch, which is in C++ but has bindings for 10 languages. But here's the thing: the "Used By" counter seems to only pick up one language at a time. So, it might show the Python users, or the Rust folks, but never everyone together.

Also, it's missing out on the C/C++ projects that are pulling in USearch as a submodule. This kind of gives an edge to the Python-like languages and doesn't really show the love for the low-level libraries that many projects lean on.

It'd be awesome if:

1. The counter could sum up users from all languages.
2. It could also catch those using the repo as a submodule.

I think these tweaks would give a better picture of how much a project is really being used. Thanks for considering this, and keep up the awesome work!
